common/Log.hpp

```diff
@@ -31,6 +31,18 @@
 
 #include "Util.hpp"
 
+enum Level
+{
+    FTL = 1, // Fatal
```

Comment:
If we must use our own enum, then we certainly want to assign the values from Poco's `Priority` enum. This is very fragile as it is.

Reply:
Again - not having the Poco header included is the end goal here; but we can have an (internal) mapping function that will evaporate away at compile time I think.

Reply:
The "enum" data type are for future internal implementation logging framework, it coincides from API POCO Message Priority, I do not think they will modify the order, maybe add new value, by the time they add something we will get rid of the dependency.